tests/integration/commands/help/__snapshots__/help.test.ts.snap

```diff
@@ -33,7 +33,8 @@ COMMANDS
   $ status        Print status information
   $ switch        Switch your active Netlify account
   $ unlink        Unlink a local folder from a Netlify site
-  $ watch         Watch for site deploy to finish"
+  $ watch         Watch for site deploy to finish
+  $ logs          Stream logs from your site"
```

Comment:
We should be more specific here on what this means

Reply:
I think this covers it? Don't want to go too in-depth on a top level command imo